docker-compose.yaml

```diff
@@ -1,19 +1,20 @@
-version: '3.9'
+version: "3.3"
+
 services:
   website:
-    build:
-      context: .
-      dockerfile: ./website/Dockerfile
-      args:
-        - HUGO_BASEURL=https://perses.dev
+    image: hugomods/hugo
+    command: hugo server --watch --bind=0.0.0.0 --disableLiveReload=true --minify -environment production --baseURL https://perses.dev --appendPort=false
     restart: unless-stopped
     ports:
-      - "5001:80"
+      - 1313:1313
+    volumes:
+      - ./website:/src
     labels:
       - "traefik.enable=true"
       # The domain the service will respond to
       - "traefik.http.routers.website.rule=Host(`perses.dev`)"
       - "traefik.http.routers.website.tls.certresolver=myresolver"
+      - "traefik.http.services.website.loadbalancer.server.port=1313"
```

Comment:
that's curious we need to add this label. I thought traefic were able to find using the docker discovery what was the port to use to send the traefic. But maybe I'm wrong

Reply:
Actually I didn't tried 😄
I can check if it's really required

Reply:
@Nexucis it works without this one.